setup.py

```diff
@@ -1,43 +1,3 @@
 import setuptools
-import pathlib
-import sys
 
-setup_reqs = ["setuptools_scm"]
-install_reqs = []
-test_reqs = [
-    "pytest",
-    "pytest-flake8",
-    "pytest-coverage",
-    "asynctest",
-    "black==20.8b1",
-    "pytest-black",
-]
-dev_requires = install_reqs + test_reqs + ["documenteer[pipelines]"]
-scm_version_template = """# Generated by setuptools_scm
-__all__ = ["__version__"]
-__version__ = "{version}"
-"""
-tools_path = pathlib.PurePosixPath(setuptools.__path__[0])
-base_prefix = pathlib.PurePosixPath(sys.base_prefix)
-data_files_path = tools_path.relative_to(base_prefix).parents[1]
-
-setuptools.setup(
-    name="ts-cbp",
-    use_scm_version={
-        "write_to": "python/lsst/ts/cbp/version.py",
-        "write_to_template": scm_version_template,
-    },
-    setup_requires=setup_reqs,
-    install_requires=install_reqs,
-    extras_require={"dev": setup_reqs + install_reqs + test_reqs + dev_requires},
-    packages=setuptools.find_namespace_packages(where="python"),
-    package_dir={"": "python"},
-    package_data={"": ["*.rst", "*.yaml"]},
-    scripts=["bin/run_cbp.py"],
-    tests_require=test_reqs,
-    license="GPL",
-    project_urls={
-        "Bug Tracker": "https://jira.lsstcorp.org/secure/Dashboard.jspa",
-        "Source Code": "https://github.com/lsst-ts/ts_CBP",
-    },
-)
+setuptools.setup()
```

Comment:
I think the idea is that you can remove setup.py completely unless you absolutely need it to exist.